### PR TITLE
Fix typos - Readme

### PR DESCRIPTION
**Pull request by herrerameri**
Hi @ilanolkies ! :wave: 
Please review it when you've a while.
Kiss!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ You will also see any lint errors in the console.
 ### `npm test`
 
 Launches the test runner in the interactive watch mode.<br>
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+See this section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
```
Comment (ilanolkies) [inline README.md]: This text should be as it was.

File: README.md [hunk 2]
```diff
@@ -65,7 +65,7 @@ It correctly bundles React in production mode and optimizes the build for the be
 The build is minified and the filenames include the hashes.<br>
 Your app is ready to be deployed!
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+See this section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
```
Comment (ilanolkies) [inline README.md]: This text should be as it was.

File: README.md (hunk 1 above)
Comment (herrerameri) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (herrerameri) [reply README.md]: Done

**Pull request merged**